app/views/core/SingletonAppVueComponentView.js

```diff
@@ -0,0 +1,27 @@
+import VueComponentView from './VueComponentView'
+
+import store from 'core/store'
+import cocoVueRouter from 'core/CocoVueRouter'
+
+import Root from './Root'
+
+export default class SingletonAppVueComponentView extends VueComponentView {
```

Comment:
Do we need this new view, or can we modify `VueComponentView` to handle vue router?

Reply:
Added a wrap up task to consolidate this as discussed https://app.asana.com/0/1115428071773315/1118364384628027/f